### ISSUE TITLE
Globe poster image/fallback

### ISSUE THREAD

**Issue by gmzbri** [vasturiano/globe.gl#146]
Hello,

Would it be possible to set a static poster image of the globe? Sometimes, on slow connection, the globe takes a couple of seconds to appear and would be nice to have an static version of the globe for these waiting seconds so it's not empty.
I am not sure if that's related with having the _waitForGlobeReady_ on?

Thanks a lot for the work put on this.

**Comment by vasturiano** [comment]
@gmzbri if the wait on seeing the globe is due to the bandwidth of loading a high-resolution image, what you can do is load first a low-res image so it appears quicker, and afterwards the intended hi-res one.

`waitForGlobeReady` just defines whether the visualization layers should appear or not before the globe image is loaded.
